src/main/cli/commandLine.ts

```diff
@@ -0,0 +1,120 @@
+                const publication = await catalogService.importFile(path as string);
+                const store = container.get("store") as Store<RootState>;
+                if (publication && notRead) {
+                    store.dispatch({
+                        type: readerActions.ActionType.OpenRequest,
+                        payload: {
+                            publication: {
+                                identifier: publication.identifier,
+                            },
+                        },
+                    });
+                    notRead = false;
+                }
+                if (path !== "." && !publication && notRead) {
+                    console.error("Publication error, path:", path);
+                }
+            });
+        },
+        help: [],
+    },
+    {
+        name: "import",
+        fct: async ({ argv }) => {
+            const catalogService = container.get("catalog-service") as CatalogService;
+            return await catalogService.importFile(argv.import as string);
+        },
+        help: [
+            "--import PATH",
+            "import epub or lpcl file",
+        ],
+    },
+    {
+        name: "opds",
+        fct: async ({ argv }) => {
+            // extract and save the title and url from opdsFeed
+            // title=http://myurl.com or get TLD and set url
+            const feed = (argv.opds as string).split("=");
+            const url = feed.length === 2 ? feed[1] : feed[0];
+            const title = feed.length === 2 ? feed[0] : extractHostname(url, true);
+            const opdsRepository = container.get("opds-feed-repository") as OpdsFeedRepository;
+            return await opdsRepository.save({ title, url });
+        },
+        help: [
+            "--opds URL",
+            "import opds feed url",
+        ],
+    },
+    {
+        name: "silent",
+        fct: async (param) => {
+            param.quit = true;
+        },
+        help: [
+            "--silent",
+            "stay on command line and don't open main window",
+        ],
+    },
+    {
+        name: "read",
+        fct: async ({ argv }) => {
+            // read the publication name
+            const publicationRepo = container.get("publication-repository") as PublicationRepository;
+            const publication = await publicationRepo.searchByTitle(argv.read as string);
+            if (publication && publication.length) {
+                const store = container.get("store") as Store<RootState>;
+                store.dispatch({
+                    type: readerActions.ActionType.OpenRequest,
+                    payload: {
+                        publication: {
+                            identifier: publication[0].identifier,
+                        },
+                    },
+                });
+            }
+        },
+        help: [
+            "--read STRING",
+            "open the reader on your publication",
```

Comment:
Suggestion - more explicit message:
"searches already-imported publications with the provided TITLE, and opens the reader with the first match"